kin/blockchain/builder.py

```diff
@@ -6,16 +6,14 @@
 from stellar_base.keypair import Keypair
 from stellar_base.memo import NoneMemo
 
-from .horizon import HORIZON_LIVE, HORIZON_TEST
-from .horizon import Horizon
 from .utils import is_valid_address, is_valid_secret_key
 
 
 class Builder(BaseBuilder):
```

Comment:
what does it build? transaction? class name is not clear

Reply:
Thats from David's code, not mine, its a transaction builder, and I assume the name is the same since its kind of an override of the Builder class from py-stellar-base